README.md

```diff
@@ -450,7 +450,7 @@ task_definition:
   task_execution_role: string            // Needed to use Cloudwatch Logs or ECR with your ECS tasks
   task_size:                             // Required for running tasks with Fargate launch type
     cpu_limit: string
-    mem_limit: string
+    mem_limit: string                    // Number-only values default to MiB
```

Comment:
Hmm, "number-only" is a bit confusing to me, esp since the value type is a string. What if we said something more like "values specified without units default to number of bytes in MiB"?